inc/class-licensing.php

```diff
@@ -220,8 +220,8 @@ public function doLicense( $metadata, $post_id = 0, $title = '' ) {
 			return '';
 		}
 
-		// Unless section is licensed differently, it should display the book license statement
-		if ( $section_license === $book_license || empty( $section_license ) ) {
+		// Unless section is empty, it should display the book license statement.
+		if ( empty( $section_license ) ) {
```

Comment:
If you do git blame (git-annotate) you'll see that this line is linked to this issue:

> Unless chapter is licensed differently, it should display the book license statement 
https://github.com/pressbooks/pressbooks/issues/1508

We want to preserve that fix. 

They key part of the new bug is:

> Chapter license statement reflects only attribution for that specific chapter ***when a chapter author has been defined.***
https://github.com/pressbooks/pressbooks-book/issues/588

Probably (untested):

```php
if (
	( $section_license === $book_license && empty( $section_author ) )
	|| empty( $section_license )
) {
```